### ISSUE TITLE
Calling identityWidget.init() doesn't seem to validate the token expiration

### ISSUE THREAD

**Issue by richtera** [netlify/netlify-identity-widget#146]
I see several instances where init() is called on('init') is called back with a valid user yet the actual underlying jwt token is expired when I do user.jwt().then() I can an expired token the backend will reject. Not I know that I can manually validate the token get it's expiration time, but it seems to me that the init() function should do this validation. Does this make sense or is this not a normal use case?

**Comment by Offirmo** [comment]
Yes. My token expired this morning, I had to logout / login to get a fresh one. We should have a way to handle that!

**Comment by joshuaferr1s** [comment]
The time I see this most happen with my application is when I push a new version (there could be a correlation between my time between commits and JWT expiry) I get the problem of init giving me the user with an expired JWT so the backend rejects it
